libbeat/autodiscover/providers/docker/docker.go

```diff
@@ -136,23 +136,24 @@ func (d *Provider) emitContainer(event bus.Event, flag string) {
 	}
 
 	meta := common.MapStr{
-		"container": common.MapStr{
-			"id":     container.ID,
-			"name":   container.Name,
-			"image":  container.Image,
-			"labels": labelMap,
+		"id":   container.ID,
+		"name": container.Name,
+		"image": common.MapStr{
+			"name": container.Image,
 		},
+		// TODO: Do we need some dedotting here?
+		"labels": labelMap,
```

Comment:
We don't need dedotting here for autodiscover because the autodiscover events are for internal use, will not store into ES. @jsoriano Please correct me if I'm wrong. Thanks!